packages/ui-kit/src/helpers/graphql/script.cjs

```diff
@@ -2,7 +2,10 @@ const fs = require('node:fs')
 
 const generatedFilePath = process.argv[2]
 const generatedFileContent = fs.readFileSync(generatedFilePath).toString()
+
 fs.writeFileSync(
   generatedFilePath,
-  generatedFileContent.replaceAll(/import { requestinit } from.+\r?\n/gi, '')
+  generatedFileContent
+    .replaceAll(/import { requestinit } from.+\r?\n/gi, '')
+    .replaceAll(/ variables]/gi, ' dataSource, variables]')
```

Comment:
If we have a ticket open with GraphQL Code Generator, can you please include it in a comment here?